docs/core/tools/dotnet-nuget-trust.md

```diff
@@ -0,0 +1,319 @@
+
+- **`-h|--help`**
+
+  Prints out a description of how to use the command.
+
+- **`-v, --verbosity <LEVEL>`**
+
+  Set the verbosity level. Defaults to m[inimal]. Allowed values are q[uiet], m[inimal], n[ormal], d[etailed], and diag[nostic].
+
+### `author`
+
+Adds a trusted signer with the given name, based on the author signature of the package.
+
+#### Synopsis
+
+```dotnetcli
+dotnet nuget trust author <name> <package> [--allow-untrusted-root] [--configfile <PATH>] [-h|--help] [-v, --verbosity <LEVEL>]
+```
+
+#### Arguments
+
+- **`name`**
+
+  The name of the trusted signer to add. If `name` already exists in the configuration, the signature is appended.
+
+- **`package`**
+
+  The given `package` should be a local path to the signed *.nupkg* file.
+  
+#### OPTIONS:
+
+- **`--allow-untrusted-root`**
+
+  Specifies if the certificate for the trusted signer should be allowed to chain to an untrusted root. This is not recommended.
+
+- **`--configfile <PATH>`**
+
+  Specific NuGet file to use instead of the standard hierarchy.
+
+- **`-h|--help`**
+
+  Prints out a description of how to use the command.
+
+- **`-v, --verbosity <LEVEL>`**
+
+  Set the verbosity level. Defaults to m[inimal]. Allowed values are q[uiet], m[inimal], n[ormal], d[etailed], and diag[nostic].
+
+### `repository`
+
+Adds a trusted signer with the given name, based on the repository signature or countersignature of a signed package.
+
+#### Synopsis
+
+```dotnetcli
+dotnet nuget trust repository <name> <package> [--allow-untrusted-root] [--configfile <PATH>] [-h|--help] [-v, --verbosity <LEVEL>]
+```
+
+#### Arguments
+
+- **`name`**
+
+  The name of the trusted signer to add. If `name` already exists in the configuration, the signature is appended.
+
+- **`package`**
+
+  The given `package` should be a local path to the signed *.nupkg* file.
+
+#### OPTIONS:
+
+- **`--allow-untrusted-root`**
+
+  Specifies if the certificate for the trusted signer should be allowed to chain to an untrusted root. This is not recommended.
+
+- **`--configfile <PATH>`**
+
+  Specific NuGet file to use instead of the standard hierarchy.
+
+- **`-h|--help`**
+
+  Prints out a description of how to use the command.
+
+- **`-v, --verbosity <LEVEL>`**
+
+  Set the verbosity level. Defaults to m[inimal]. Allowed values are q[uiet], m[inimal], n[ormal], d[etailed], and diag[nostic].
+
+### `certificate`
+
+Adds a trusted signer with the given name, based on a certificate fingerprint.
+
+#### Synopsis
+
+```dotnetcli
+dotnet nuget trust certificate <name> <fingerprint> [--algorithm <ALGORITHM>] [--allow-untrusted-root] [--configfile <PATH>] [-h|--help] [-v, --verbosity <LEVEL>]
+```
+
+#### Arguments
+
+- **`name`**
+
+  The name of the trusted signer to add. If a trusted signer with the given name already exists, the certificate item is added to that signer. Otherwise a trusted author is created with a certificate item from the given certificate information.
+
+- **`fingerprint`**
```

Comment:
```suggestion
- **`NAME`**

  The name of the trusted signer to add. If a trusted signer with the given name already exists, the certificate item is added to that signer. Otherwise a trusted author is created with a certificate item from the given certificate information.

- **`FINGERPRINT`**
```